src/components/Dropdown/asyncDropdown/AsyncDropdown.jsx

```diff
@@ -189,6 +209,8 @@ class AsyncDropDown extends React.Component {
                 onCreate={onCreate && handleCreate}
                 customCreateButtonText={customCreateButtonText}
                 hideTotalSearchCount={hideTotalSearchCount}
+                page={this.state.page}
```

Comment:
gjentar meg selv med `const { page } = this.state;` bare så du er klar over at det _kan_ endres her og